services/src/main/java/org/keycloak/broker/saml/SAMLIdentityProvider.java

```diff
@@ -130,6 +130,9 @@ public Response performLogin(AuthenticationRequest request) {
                 requestedAuthnContext.addAuthnContextDeclRef(authnContextDeclRef);
 
             String loginHint = getConfig().isLoginHint() ? request.getAuthenticationSession().getClientNote(OIDCLoginProtocol.LOGIN_HINT_PARAM) : null;
+            Boolean allowCreate = null;
+            if ( getConfig().getConfig().get("allowCreate")== null ||  getConfig().isAllowCreate())
```

Comment:
Prefer constants to literals:
```suggestion
            if (getConfig().getConfig().get(ALLOW_CREATE) == null || getConfig().isAllowCreate())
```